### ISSUE TITLE
Svelte Find File References creates .vscode/settings.json file

### ISSUE THREAD

**Issue by kilicbaran** [sveltejs/language-tools#1516]
### Describe the bug

Svelte Find File References creates a file named .vscode/settings.json containing an empty JSON object.

### Reproduction

1. Right click on a Svelte component
2. Choose `Svelte: Find File References`
3. There is an unexpected file `.vscode/settings.json`

### Expected behaviour

The file `.vscode/settings.json` should not be created if it doesn't exist already.

### System Info

- OS: macOS
- IDE: VSCode

### Which package is the issue about?

Svelte for VS Code

**Comment by Jojoshua** [comment]
I don't think this behavior is unique to the svelte extension, the same behavior applies if using Find File References on a .ts file.

**Comment by kilicbaran** [comment]
Then, it is a VSCode bug not extension bug. Thanks @Jojoshua !
